registry/donor/views.py

```diff
@@ -111,6 +111,9 @@ def overview_data():
 def detail(rc):
     remove_medal_form = RemoveMedalForm()
     overview = DonorsOverview.query.get(rc)
+    if overview is None:
+        return render_template('404.html'), 404
```

Comment:
```suggestion
        return render_template("404.html"), 404
```

Neprochází linterem.

Reply:
fixed it